global.bicep

```diff
@@ -0,0 +1,2 @@
+/** The isolated subscription for testing */
```

Comment:
I would suggest putting this somewhere under eng folder instead of in the root of the repo.